### DISCUSSION TITLE
离群点

### DISCUSSION THREAD

**Discussion by giscus** [ouuan/iles-blog#301] [Comments]
# 离群点

身处小众神秘专业，贵系课一半 A+，GPA 3.45，躲开最热门的 AI，挑战无人问津的的课程项目，跳过打杂阶段独立进行科研，然而保研前 0 pub……
没有参考价值，不能作为经验分享，一个离群点的故事。

https://ouuan.moe/post/2024/07/outlier

**Comment by leverimmy** [comment]
假如你是欧拉

**Reply by ouuan** [reply]
~~假如我是欧拉（指正（~~

**Reply by leverimmy** [reply]
~~因为你会说“假如我是欧拉”，所以我应说“假如你是欧拉”（严谨~~

**Comment by LeSnow-Ye** [comment]
膜

**Comment by Sweetlemon68** [comment]
后记一段给我一种史诗般的震撼。适合在我成日摸鱼的每个白天和晚上阅读。

**Comment by namasikanam** [comment]
天呐，读了之后想起自己的本科生活了！一点碎碎念：

1. 哇计网联合试验现在都声名远扬了嘛！我当初也是完全没碰 CPU，导致我计原基本啥也不会哈哈。
2. 天呐你竟然也做了 formal verification for OS 作为大作业，竟然也搞得是 memory allocator，是不是被我带跑了哈哈。我当时千挑万选在某个 OS 里面挑出了 page allocator 这么一个比较好加性质也比较好证的模块哈哈。
